javascript/javascript1/week3/note_taking.js

```diff
@@ -0,0 +1,43 @@
+const notes=[];
+//Save or adding a note
+function addNote(content, id){
+    const objectNote={};
+      objectNote.content=content;
+    objectNote.id=id;
+    notes.push(objectNote);
+}
+addNote("Some methodes that help to solve the problem", 13);
```

Comment:
It's better to put this code after all the function declarations. I almost missed it